### PR TITLE
Contact form PR

### PR DESCRIPTION
**Pull request by til2to**
## Added Contact form using HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
...
+        </p>
+      </header>
+        <form action="https://formspree.io/f/xrgdbeyr" method="post">
+          <fieldset>
+          <input type="text" id="name" name="Name" placeholder="Text Input" required
+          maxlength="30"
+          />
+          <input type="email" id="email" name="Email" placeholder="jeremy@gmail.com" required
+          />
+          <textarea name="Text Area" id="textarea" placeholder="Enter text here" required></textarea>
```
Comment (mugishasam123) [inline index.html]: - [x] Kindly add validations to your textarea to make it more professional. this textarea should have` maxlength="500" `attribute. you got this 👍

File: index.html [hunk 2]
```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
   
       <section class="button">See Project</section>
     </section>
-
+    <section id="contact-me">
+      <header>
+        <h2>CONTACT US</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
```
Comment (mugishasam123) [inline index.html]: - [ ] Kindly stick to the Figma design as much possible  to Make your project looks good. you can do so by enhancing colors ,font-sizes,font-weight,etc.

Figma design:
image

Your design:
image

File: index.html [hunk 3]
```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
   
       <section class="button">See Project</section>
     </section>
-
+    <section id="contact-me">
+      <header>
```
Comment (mugishasam123) [inline index.html]: - [x] Please you should not use the `header ` tag in the other section beside the navbar section, to make your code more semantically correct I would recommend you remove it.

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @til2to  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
-  No linter errors 👍 
- Good commits 🚀 
- Nice use of git-flow 🥇 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ Optional]  I recommend you wrap all images in one folder to make your project structure more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @til2to ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Great Design✔️
- Form Validations working perfectly ✔️
- Form Submission works perfectly ✔️


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**